<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExoMars2020</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_log_address">
           <property name="text">
            <string>Logical address (value from 32 to 254) :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="log_address_sb">
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>254</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_fsize">
           <property name="text">
            <string>Size of frames in bit (value from 1 to 16) :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fsize_sb">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>16</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_speed_bytes">
           <property name="text">
            <string>Speed of sending bytes (units in abbreviated form) :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="speed_bytes_line">
           <property name="placeholderText">
            <string>Time with units</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_speed_channel">
           <property name="text">
            <string>Speed of the channel (units in abbreviated form) :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="speed_channel_line">
           <property name="placeholderText">
            <string>Time with units</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_error_period">
           <property name="text">
            <string>Error period (units in abbreviated form) :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="error_period_line">
           <property name="placeholderText">
            <string>Time with units</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="setNodecarac">
         <property name="text">
          <string>Set caracteristics</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="add">
     <property name="text">
      <string>Add Node</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete_2">
     <property name="text">
      <string>Delete Node</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
